### ISSUE TITLE
released whl package for different spark version, like 2.1, 3.0, etc.

### ISSUE THREAD

**Issue by qiuxin2012** [intel-analytics/analytics-zoo#4017]

**Comment by glorysdj** [comment]
https://github.com/intel-analytics/analytics-zoo/blob/master/pyzoo/dev/release_default_linux.sh

need to modify this script and can pass spark version to it
and also modify jenkins job:
http://172.168.2.101:8080/view/ZOO-Release/job/ZOO-Release-PyPI/configure
and also upload releases to sourceforge

@pinggao187 can you fix this

**Comment by hkvision** [comment]
> need to modify this script and make it can accept spark version
> https://github.com/intel-analytics/analytics-zoo/blob/master/pyzoo/dev/release_default_linux.sh
> and also modify jenkins job:
> http://172.168.2.101:8080/view/ZOO-Release/job/ZOO-Release-PyPI/configure
> and also upload releases to sourceforge
> 
> @pinggao187 can you fix this

The main issue is how to release different packages under the same project?

**Comment by glorysdj** [comment]
> > need to modify this script and make it can accept spark version
> > https://github.com/intel-analytics/analytics-zoo/blob/master/pyzoo/dev/release_default_linux.sh
> > and also modify jenkins job:
> > http://172.168.2.101:8080/view/ZOO-Release/job/ZOO-Release-PyPI/configure
> > and also upload releases to sourceforge
> > @pinggao187 can you fix this
> 
> The main issue is how to release different packages under the same project?

if we can not release different versions to same pypi project, we can upload different spark versioned artifacts on sourceforge.

**Comment by hkvision** [comment]
yes. we can first upload whls to sourceforge. Maybe need to change the whl name to add spark version.

**Comment by pinggao187** [comment]
[released whl package for different spark version #4045](https://github.com/intel-analytics/analytics-zoo/pull/4045)
